app/controllers/api/v2/host_reports_controller.rb

```diff
@@ -60,7 +61,9 @@ def create
             keywords, unique_by: :name
           ).rows.flatten
         end
-        process_response @host_report.save
+        result = @host_report.save
+        @host_report.body = nil
```

Comment:
Maybe it's better to just update the .rabl template for create action by removing :body attribute from there?

Reply:
But then I need to break the nice and consistent create.rabl extends show.rabl. Is there a nice way to do that?

Reply:
Unfortunatelly I didn't find a nicer way than just to swap extends so in `show.rabl` you'd have

```ruby
# frozen_string_literal: true

object @host_report

extends 'api/v2/host_reports/create'

attributes :body
```
and in `create.rabl`

```ruby
# frozen_string_literal: true

object @host_report

extends 'api/v2/host_reports/main'

attributes :status

node :keywords do |report|
  report.report_keywords.map(&:name)
end
```

I guess `:status` can also go to the `main.rabl`

Reply:
Isn't keeping this single line better? That refactoring might introduce bugs, in case we add a new attribute it could be missing from one or another.